script/build_apm_docs.sh

```diff
@@ -17,16 +17,12 @@ else
     git -C $docs_dir pull https://github.com/elastic/docs.git
 fi
 
-index_list="$(find ${GOPATH%%:*}/src/$path -name 'index.asciidoc')"
```

Comment:
Ill summon @bmorelli25 for this - i sort of recall the case of building the whole apm docs to test outgoing links, for which this loop was needed. I don't know if that is still a valid use case...

Reply:
Now that we have link checking in our PRs, this is fine to remove.